src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -2780,4 +2781,18 @@ public void sendMessage(@NotNull ChatMessageType position, @NotNull BaseComponen
 
 	}
 
+	@Override
+	public void setFlyingFallDamage(@NotNull TriState flyingFallDamage)
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
+	@Override
+	public @NotNull TriState hasFlyingFallDamage()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
```

Comment:
Can these be easily implemented?

Reply:
I don't know, maybe. But isn't that part of another pull request? 

(I can implement them if it's fine in this one)